src/runtime_stack_check.js

```diff
@@ -14,8 +14,8 @@ function writeStackCookie() {
   // The stack grow downwards towards _emscripten_stack_get_end.
   // We write cookies to the final two words in the stack and detect if they are
   // ever overwritten.
-  {{{ makeSetValue('max', 0, '0x2135467', 'i32' ) }}};
-  {{{ makeSetValue('max', 4, '0x89BACDFE', 'i32' ) }}};
+  {{{ makeSetValue('max', 0, '0x2135467', 'i32', 0, true) }}};
+  {{{ makeSetValue('max', 4, '0x89BACDFE', 'i32', 0, true) }}};
```

Comment:
How about instead we just change these lines to:

```
HEAPU32[(max >> 2)] = 0x2135467
HEAPU32[(max >> 2) + ] = 0x89BACDFE
```

(also the line below would need updating too).

Reply:
even better you can do:

```
var max = _emscripten_stack_get_end() >> 2;
HEAPU32[max] = 0x2135467;
HEAPU32[max + 1] = 0x89BACDFE;
```